Server/bkr/server/model/activity.py

```diff
@@ -53,9 +53,9 @@ def __init__(self, user=None, service=None, action=None,
         self.action = action
 
         if old_value is not None:
-            old_value = unicode(old_value)[:old_value_max_length]
+            old_value = u"{}".format(old_value)[:old_value_max_length]
```

Comment:
u"" doesn't mean anything in Python 3. In fact they had dropped it at one time and then put it back in.

Are we sure this is needed?

Reply:
Well as I said. I'm still running it in Python 2. This is actually really good that it doesn't affect Python 3 - that's the reason why I put it here like this.

Reply:
Makes sense. I'll be very happy when we stop doing Python 2, but until then...